packages/plugin-react/src/index.ts

```diff
@@ -14,10 +13,11 @@ import {
   silenceUseClientWarning,
   virtualPreamblePlugin,
 } from '@vitejs/react-common'
-import {
-  exactRegex,
-  makeIdFiltersToMatchWithQuery,
-} from '@rolldown/pluginutils'
+import { readFileSync } from 'node:fs'
+import { dirname, join } from 'node:path'
+import { fileURLToPath } from 'node:url'
+import { createFilter } from 'vite'
```

Comment:
I'm note a fan of having all type import before runtime ones, it create more diffing when switching from type import to runtime or the other way around and when importing both types and import (I personally like them to be merged) it's cleaner to have them next to each other IMO if not merged